editor/editor_file_system.cpp

```diff
@@ -43,6 +43,7 @@
 #include "editor/editor_resource_preview.h"
 #include "editor/editor_settings.h"
 #include "editor/project_settings_editor.h"
+#include "plugins/script_editor_plugin.h"
```

Comment:
We prefer to use "absolute" include paths for consistency.
```suggestion
#include "editor/plugins/script_editor_plugin.h"
#include "editor/project_settings_editor.h"
```

Reply:
Wow, I copy/paste from the only place in all Godot code base where the absolute path was not used for `script_editor_plugin.h`!!
I changed it in `C:\Projects\godot\editor\connections_dialog.cpp` also.